### DISCUSSION TITLE
Block quote element in RTE?

### DISCUSSION THREAD

**Discussion by christian-reichart** [payloadcms/payload#1305] [Roadmap]
Hey there,
when creating a Rich Text field I have the option to allow 'blockquote' inside the admin elements.

However, the editor doesn't pick up anything and I can't find this element in the docs. https://payloadcms.com/docs/fields/rich-text

Is this a left over? Are there any plans to include a blockquote in the core?

Thanks for any hints :)

**Comment by jacobsfletch** [comment]
Hi @christian-reichart, yup I see that in the docs we render a `quote` in the example `serialize` function, even though blockquote is not identified as being a core RTE element. [Here's the source code](https://github.com/payloadcms/payload/blob/9f6f2d24f3a417ad10b8483324c361750a5c0aae/src/admin/components/forms/field-types/RichText/RichText.tsx#L29). The default elements are h1, h2, h3, h4, h5, h6, link, ol, ul, indent, relationship, and upload as indicated in the docs. So we should either remove those lines from the example or add core support.

**Comment by JarrodMFlesch** [comment]
Was added in [this PR](https://github.com/payloadcms/payload/pull/2533).
